### ISSUE TITLE
UI Owners

### ISSUE THREAD

**Issue by Kapautza** [covoucher/business#10]
Minor UI issues

Sign-up: Country pre-selected should be according to IP address, see #9.

On the "Connect your account to Stripe" sign-up page: 

- I would rephrase the statement about Stripe: "We collaborate with Stripe for processing payments. This ensures that your bank details will be secure and that you will be paid instantly after a purchase has been made. Click  below...." 

- Button should be smaller and centred on that page, see issue #9.
 
When connecting to Stripe:

- Why do we need industry? I like it as we can have a filter on the homepage where patrons can filter for what they want to buy (or by location). But do we want to make it mandatory?

- I selected a standard German code and this doesn't work. Also a UK one.  Now I understand that it's filtered by country. Perhaps -- given that we allow to use a private address etc -- we allow to change the country here? Not sure, we should discuss this.

- When adding directors: Several persons can have the same email address. Is that correct?

TBC crucial next issue is fixed.

**Comment by mm3509** [comment]
I included the text about Stripe.
